<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>doubleDialog</class>
 <widget class="QDialog" name="doubleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="xlineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>102</x>
     <y>100</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>15</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>请输入文字</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ylineEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前位置坐标</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>190</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tlineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>1</number>
   </property>
   <property name="placeholderText">
    <string>请输入持续时间</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
